addusers.py

```diff
@@ -257,6 +246,30 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
+    
+    help_description = ("Add new users and projects to OpenStack using " 
+                        "data from Google Sheets.")
+    parser = argparse.ArgumentParser(description=help_description)
+    parser.add_argument('-c', '--config', 
+                        help='Specify configuration file.')
+
+    args = parser.parse_args()
+   
+    if args.config is not None:
+        CONFIG_FILE = set_config_file(args.config)
+    else:
+        CONFIG_FILE = set_config_file()
+
+    config = ConfigParser.ConfigParser()
+    config.read(CONFIG_FILE)
+
+    admin_user = config.get('auth', 'admin_user')
+    admin_pwd = config.get('auth', 'admin_pwd')
```

Comment:
It is a little disconcerting that the admin user and password has to be included configuration file.  I am OK with this if it is the only way to do this, however, this probably would not pass a security audit.